src/events/http.js

```diff
@@ -305,14 +305,11 @@ function httpWrapper(wrappedFunction) {
                     }
                 });
 
-                clientRequest.on('close', () => {
-                    resolveHttpPromise(httpEvent, resolve, startTime);
-                });
-
                 clientRequest.on('response', (res) => {
-                    res.on('data', (chunk) => { data += chunk; });
+                    res.on('data', (chunk) => { chunks.push(chunk); });
```

Comment:
what happens if the response is too large? e.g. a 10gb movie ? (not that we have those :-)

Reply:
You're right. Before we send the trace we make sure it does not cross the limit (64KB if I recall correctly), and if so we remove such fields.

Reply:
I think @tripodsan is right, and we should add some limit here (first 5 chinks?) To not use too much memory

Reply:
👍 Done